SportsHub/UnitTests/MockData/UserLoginDTOMockData.cs

```diff
@@ -0,0 +1,24 @@
+﻿using SportsHub.AppService.Authentication.Models.DTOs;
+
+namespace UnitTests.MockData;
+
+public class UserLoginDTOMockData
+{
+    public static UserLoginDTO GetUserWithUsername()
+    {
+        return new UserLoginDTO()
+        {
+           UsernameOrEmail = "gogo",
+           Password = "testpassword123"
+        };
+    }
+
+    public static UserLoginDTO GetUserWithEmail()
+    {
+        return new UserLoginDTO()
+        {
```

Comment:
Maybe better to use one method and add UserNameorEmail in the parameter? Do we really need a separate class for it?

Reply:
I will change it .